certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3050 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "InEight",
+    "description": "The InEight Connector provides an API to work with InEight objects",
```

Comment:
@InEightDevOps Please provide a detailed description of your connector which should describe concisely the main purpose and value offered by your connector and can be understandable by everyone.

Docs - [Write a description for your connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)